app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,31 @@
+%h2 Register
+
+= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
```

Comment:
You can make use of the new `form_with` helper

```
= form_with(model: resource, as: resource_name, url: registration_path(resource_name), remote: false) do |f|

```

Reply:
We have been searching online and we do not see how it would benefit our code, could you please explain this new helper method to us?